geode-core/src/integrationTest/java/org/apache/geode/logging/internal/ConfigurationInfoIntegrationTest.java

```diff
@@ -12,19 +12,17 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package org.apache.geode.internal.logging;
+package org.apache.geode.logging.internal;
 
-import static org.apache.geode.internal.logging.ConfigurationInfo.getConfigurationInfo;
+import static org.apache.geode.logging.internal.ConfigurationInfo.getConfigurationInfo;
 import static org.assertj.core.api.Assertions.assertThat;
 
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 
 import org.apache.geode.test.junit.categories.LoggingTest;
 
-/**
- * Integration tests for {@link ConfigurationInfo}.
- */
+/** Integration tests for {@link org.apache.geode.logging.internal.ConfigurationInfo}. */
```

Comment:
Fully qualified?

Reply:
Same

Reply:
I pushed more fixes to the javadocs on all of these tests/